main.js

```diff
@@ -159,11 +195,18 @@ class ContentReviewLog {
      *                            embed formatter.
      */
     _processPage(page) {
-        const title = page.page_title,
-              rev = page.latestRevision.revisionId,
-              status = page.latestRevision.statusKey,
-              liveRev = page.liveRevision.revisionId,
-              curr = this._data[title];
+        const title = page.title;
+        const status = page.status;
+        const rev = page.latestRevision;
+        const liveRev = page.liveRevision;
+        const curr = this._data[title];
+
+        // Save to cache
+        this._data[title] = {
+            rev,
+            status
+        };
```

Comment:
Wouldn't this save the data to cache even when memcache screws up?

Reply:
When the cached and current values don't match up, the `shouldSave` flag's now set to false and it's not saved

Should be resolved now